server/py/services/api/tests/unit/db/test_model_endpoint.py

```diff
@@ -213,10 +213,10 @@ def test_batch_insert_and_update(self) -> None:
             self._db_session,
             "project-1",
             {
-                uuid.UUID("5cfeed66-72cc4d978ff9b7b06ebe77f2"): {
+                uuid.UUID("5cfeed6672cc4d978ff9b7b06ebe77f2").hex: {
```

Comment:
Isn't this just
```suggestion
                "5cfeed6672cc4d978ff9b7b06ebe77f2"): {
```
?